src/containers/profiles/EditProfiles/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import M from 'materialize-css';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import avatar2 from '../../../assets/images/avatar2.png';
+import { editUserProfileAction } from '../store/profileActions';
+import Modal from '../../../components/Modal';
+import './EditProfiles.scss';
+
+class EditProfiles extends Component {
+  state = {
+    bio: '',
+    image: avatar2,
+  };
+
+  componentDidMount = () => {
+    const modals = document.querySelector('.modal');
```

Comment:
Please explain the what this statement does

Reply:
It allows one to open the modal using a trigger (in this case the edit button). This was the resource used: https://materializecss.com/modals.html.